network-api/networkapi/highlights/migrations/0007_nullify_homepage.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-02-08 22:05
+from __future__ import unicode_literals
+
+from django.db import migrations
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('highlights', '0006_auto_20180208_2205'),
+    ]
+
+    operations = []
```

Comment:
what is the purpose of this migration?

Reply:
it allows safe modification of 006 to remove mention of mezzanine